frontend/src/container/PublicProfile.jsx

```diff
@@ -426,6 +482,85 @@ export class PublicProfile extends React.Component {
             />
           )}
 
+          {state.userCall && state.userCall.state === USER_CALL_STATE.IN_PROGRESS && (
+            <CardPopup
+              customClass=''
+              customHeaderClass='primaryColorBg'
+              onClose={this.handleClickCancelButton}
+              label={props.t('Call in progress...')}
+              faIcon='fas fa-phone'
+            >
+              <div className='gallery__delete__file__popup__body'>
+                <div>{props.t('{{username}} has received your call. If accepted, the call will open automatically.', { username: props.user.username })}</div>
+                <br />
+                <div className='gallery__delete__file__popup__body__btn'>
+                  <IconButton
+                    onClick={this.handleClickCancelButton}
+                    text={props.t('Cancel the call')}
+                    icon='fas fa-phone-slash'
+                  />
+
+                  <IconButton
+                    intent='primary'
+                    mode='light'
+                    onClick={props.onClickDisplayCallPopup}
+                    dataCy='gallery__delete__file__popup__body__btn__delete'
```

Comment:
Seems long and unrelated to gallery. I suggest to drop this if it's not yet used.

```suggestion
```